src/backend/bs_srcserver

```diff
@@ -6259,6 +6259,18 @@ sub cloudupload_log {
   return undef; # always streams result
 }
 
+# only to support frontend testing
+sub resetall {
+   my ($cgi) = @_;
+
+   system("rm -rf $BSConfig::bsdir/sources");
+   system("rm -rf $BSConfig::bsdir/build");
+   system("rm -rf $BSConfig::bsdir/trees");
+   system("rm -rf $BSConfig::bsdir/events");
+   system("rm -rf $BSConfig::bsdir/projects");
```

Comment:
@mlschroe @M0ses I assume you can do this better :)

Reply:
I'm looking forward to this going wrong on a production system for the first time ;-)

Reply:
Yeah, this needs to be protected by a command line option only passed in start_development_backend

Reply:
You're adopting code to the test suite again...

Reply:
Well, sure. I find this the nicer alternative to breaking the test suite. It would of course all be easier if database_cleaner would remove the underlying sources itself :)